pop-cli-for-smart-contracts/guides/calling-your-contract.md

```diff
@@ -2,14 +2,30 @@
 
 Now that your contract is deployed locally, we can start interacting with it.
 
-Grab the contract address that was outputted from the [previous step](deploy-your-contract-locally/deploy-locally-on-a-solochain.md).
+### Interactive Guidance (Recommended)
+
+Interact with your deployed contract **with** Pop CLI's interactive guidance by simply entering:
 
 ```shell
-pop call contract -p ./flipper --contract 5CLPm1CeUvJhZ8GCDZCR7nWZ2m3XXe4X5MtAQK69zEjut36A --message get --suri //Alice
+pop call contract
 ```
 
+Grab the contract address that was outputted from the [previous step](deploy-your-contract-locally/deploy-locally-on-a-solochain.md).
```

Comment:
https://github.com/r0gue-io/pop-docs/issues/13

Reply:
I think we should stick with the existing style for consistency, at least until the refactor is done.